src/components/elements/section/types.ts

```diff
@@ -2,11 +2,17 @@ import { TComponentOmitProps, TFrontendEngineFieldSchema } from "../../frontend-
 import { IBaseElementSchema } from "../types";
 import { TWrapperSchema } from "../wrapper";
 
+type TCustomOptions = {
+	gridType?: "v2" | "v3" | undefined;
+	contentType?: "v2" | "v3" | undefined;
+};
```

Comment:
document them in storybook, explain what are they for and how they should be used